auth/src/adapter/redis-database-adapter.ts

```diff
@@ -107,6 +108,9 @@ export class RedisDatabaseAdapter extends Database {
         }
         try {
             this.database = Redis.createClient({ port: this.redisPort, host: this.redisHost });
+            if (Config.isTestingEnvironment()) {
```

Comment:
This should not be done here, but the test code should clear the DB before every test and sets up data for each test.